DQM/L1TMonitorClient/python/L1TStage2uGMTClient_cff.py

```diff
@@ -18,68 +18,69 @@
     ratioName = cms.untracked.string('mismatchRatio'),
     ratioTitle = cms.untracked.string('Summary of mismatch rates between uGMT muons and uGMT muon copy 1'),
     yAxisTitle = cms.untracked.string('# mismatch / # total'),
-    binomialErr = cms.untracked.bool(True)
+    binomialErr = cms.untracked.bool(True),
+    ignoreBin = cms.untracked.vint32(ignoreBins)                                                         
```

Comment:
```suggestion
    ignoreBin = cms.untracked.vint32()                                                         
```
(in any case this is not needed: the default is set in the fillDescriptions method of the `L1TStage2RatioClient` class)